src/main/java/core/basesyntax/model/Color.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax.model;
+
+public enum Color {
```

Comment:
The `Color` enum should define constants for different colors, such as `RED`, `GREEN`, `BLUE`, etc. This will allow figures to have a color attribute.